src/components/admonition/admonition.scss

```diff
@@ -0,0 +1,41 @@
+.admonition {
+    display: flex;
+    width: 100%;
+
+    &.danger {
+        background-color: rgb(var(--color-red-dark));
+    }
+    &.info {
+        background-color: rgb(var(--color-cyan-default));
+    }
+    &.note {
+        background-color: rgb(var(--color-gray-default));
+    }
+    &.tip {
+        background-color: rgb(var(--color-green-dark));
+    }
+    &.warning {
+        background-color: rgb(var(--color-orange-dark));
+    }
+
+    .admonition-side {
+        mix-blend-mode: screen;
```

Comment:
Very nice! I haven't seen this before. I noticed that browser support for `mix-blend-mode` on _svg_ is so-so, so we might want to just have a quick look in BrowserStack to make sure the icons don't look horrible in Safari for example.

https://developer.mozilla.org/en-US/docs/Web/CSS/mix-blend-mode#browser_compatibility

Reply:
I removed this effect due to accessibility issues. So I'm hiding these discussions.